build.bat

```diff
@@ -1,6 +1,4 @@
-cd core/src/nucleus/wasm-test && cargo +nightly-2018-10-12 build --release --target wasm32-unknown-unknown && cd ../../../../
-cd core_api/wasm-test/round_trip && cargo +nightly-2018-10-12 build --release --target wasm32-unknown-unknown && cd ../../../
-cd core_api/wasm-test/commit && cargo +nightly-2018-10-12 build --release --target wasm32-unknown-unknown && cd ../../../
+cd container_api/wasm-test && cargo +nightly-2018-10-12 build --release --target wasm32-unknown-unknown && cd ../../
```

Comment:
You can use the `--manifest-path` option to clean up these lines so they don't need the cd commands :)

Reply:
I know. I'm doing this on purpose so when it fails, you are in the failed directory.

Reply:
ah

Reply:
hmmm, this might need to revert to `--manifest-path` at some point, but i can't remember why...